src/Modules/Drawer.react.js

```diff
@@ -223,11 +255,11 @@ class Drawer extends Component {
         renderContent = React.Children.map(children, child => {
             if (_.isFunction(child.type)) {
                 return React.cloneElement(child, {
-                    closeButton: closeButton,
-                    inverse: inverse,
-                    onClose: onClose,
-                    title: title,
-                    titleTruncate: titleTruncate
+                    closeButton: this.props.closeButton,
+                    inverse: this.props.inverse,
+                    onClose: this._onClose,
+                    title: this.props.title,
+                    titleTruncate: this.props.titleTruncate,
```

Comment:
We cannot use the props value we destructured above?

Reply:
done.